node/src/rtpParametersFbsUtils.ts

```diff
@@ -494,12 +495,12 @@ export function parseRtpEncodingParameters(
 ): RtpEncodingParameters {
 	return {
 		ssrc: data.ssrc() ?? undefined,
-		rid: data.rid() ?? undefined,
+		rid: data.rid() || undefined,
```

Comment:
This is because if `rid` is empty string we want it to become `undefined`. Same in all similar changes below.